api/handlers/job.go

```diff
@@ -135,19 +167,19 @@ func (h *Job) handleOrgDelete(ctx context.Context, resourceGUID, jobGUID string)
 			jobGUID,
 			[]presenter.JobResponseError{},
 			presenter.StateProcessing,
-			orgDeletePrefix,
+			jobType,
 			h.serverURL,
 		), nil
 	} else {
```

Comment:
It would be more idiomatic to remove the else and instead make the `if err != nil` be a guard clause

Reply:
I've removed the else to hopefully make this clearer.